exec-agent/process/common.go

```diff
@@ -0,0 +1,68 @@
+package process
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const (
+	DefaultLogsPerPageLimit = 50
+)
+
+func maskFromTypes(types string) uint64 {
+	var mask uint64
+	for _, t := range strings.Split(types, ",") {
+		switch strings.ToLower(strings.TrimSpace(t)) {
+		case "stderr":
+			mask |= StderrBit
```

Comment:
If this code is in common file, then maybe it would be a good idea to put these constants here too?